src/utils.ts

```diff
@@ -187,11 +187,18 @@ export namespace Utils {
       );
     }
 
-    // verify homeDomain
-    if (homeDomain && `${homeDomain} auth` !== operation.name) {
-      throw new InvalidSep10ChallengeError(
-        "The transaction's operation key name does not match the expected home domain",
-      );
+    // verify any subsequent operations are manage data ops and source account is the server
```

Comment:
I'm slightly curious/concerned that allowing N Manage Data Ops will give room for anchors-client pairs to start using that functionality to pass non-standardized key-value pairs. I don't know how to feel about that, what do you think?

Reply:
This is an issue with the protocol change and not just the JS SDK's implementation. Can you raise this issue on the SEP-10 v2.1.0 protocol change (stellar/stellar-protocol#740) and we can discuss there?